infrastructure/monitoring/prometheus/prometheus.yml

```
@@ -0,0 +1,150 @@
+      - targets: ['grafana:3000']
+    scrape_interval: 30s
+    scrape_timeout: 10s
+    metrics_path: /api/health
+
+# Storage configuration
+storage:
+  tsdb:
+    # Retention policies
+    retention.time: 30d
+    retention.size: 10GB
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)